KH3Randomizer/Data/RandomizerService.cs

```diff
@@ -244,13 +244,17 @@ public class RandomizerService
             }
         }
 
+        public static Dictionary<DataTableEnum, Dictionary<string, Dictionary<string, string>>> GetDefaultOptions()
```

Comment:
This is a bit of a red flag for me - We should be not opening multiple of the same file to get the default options like line 332 and line 811

Reply:
It looks like those StreamReader calls on line 332 and line 811 were extra and unnecessary and I think I forgot to remove them when I switched over to calling this method. Doesn't look like those StreamReaders were being used for anything else with how things are currently, so I removed them and this should fix this